client/src/components/RatingsAndReviews/Ratings/ChartRating.jsx

```diff
@@ -2,50 +2,67 @@ import React from 'react';
 import styled from 'styled-components';
 
 const Container = styled.div`
+  padding-bottom: 10px;
+`
+
+const OuterBar = styled.div`
   width: 100%;
-  background-color: #ddd;
+  background-color: #dedede;
 `;
 
-const Bar = styled.div`
-  padding-top: 4px;
-  padding-bottom: 4px;
+const InnerBar = styled.div`
+  padding-top: 3px;
+  padding-bottom: 3px;
   width: ${props => props.size};
-  background-color: #3a3a3a;
+  background-color: black;
 `;
 
 export default function ChartRating({ratings}) {
 
   function getPercent(ratings, rate) {
-    let total = 0;
+    let max = 0;
     for (let k in ratings) {
-      total += Number(ratings[k]);
+      let count = Number(ratings[k])
+      if (count > max) {
+        max = count
+      }
     }
-    return String(Math.round(ratings[rate] / total * 100)) + '%'
+    return String(Math.round((ratings[rate] || 0) / max * 100)) + '%'
   }
 
 
   return (
-    <div style={{paddingRight: '40px'}}>
-      5 Stars
+    <>
```

Comment:
DRY: can you refactor this so it maps?

something like 

stars = ['5', '4', '3', '2', '1'];

{stars.map((star) => (
  4 Stars
      <Container>
        <Bar size={getPercent(ratings, {star})}></Bar>
        {star} Stars
        <OuterBar>
          <InnerBar size={getPercent(ratings, {star})}></InnerBar>
        </OuterBar>
      </Container>
));